walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,72 @@
 
 package com.google.sps;
 
-import java.util.Collection;
+import java.util.*;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    HashSet<String> attendees = new HashSet<>(request.getAttendees());
+    ArrayList<TimeRange> timeRangesNotAvailable = getTimeRangesNotAvailable(events, attendees);
```

Comment:
unavailableTimeSlots, to keep in sync with availableTimeSlots?

Reply:
Done